CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ Here is a template for new release sections
 - further sector individuals (#578)
 - emission factor (#581)
 - spatial regions (#585)
+- industrial process (#589)
```

Comment:
This should go to the unreleased-section. Unfortunately, this section doesn't exist yet, so we should add it. Additionally, we should make a note in the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/How-to-release-a-new-ontology-version) to do this in the release-PR next time.

Reply:
I've just included this in the wiki.

Reply:
I included it as well, so now we have it twice in the wiki. But I can just delete my sentence and then everything should be fine.